protocol/requests/share_community.go

```diff
@@ -10,8 +10,9 @@ var ErrShareCommunityInvalidID = errors.New("share-community: invalid id")
 var ErrShareCommunityEmptyUsers = errors.New("share-community: empty users")
 
 type ShareCommunity struct {
-	CommunityID types.HexBytes   `json:"communityId"`
-	Users       []types.HexBytes `json:"users"`
+	CommunityID   types.HexBytes   `json:"communityId"`
+	Users         []types.HexBytes `json:"users"`
+	InviteMessage string           `json:"inviteMessage"`
```

Comment:
This has to be `json:inviteMessage,omitempty` otherwise we'll break clients that don't send an invite message